docs/asciidoc/exportCypher.adoc

```diff
@@ -17,6 +17,17 @@ It is possible to choose between three export formats:
 * `cypher-shell`: for Cypher shell
 * `plain`: doesn't output begin / commit / await just plain Cypher
 
+You can also use the Optimizations like: `useOptimizations: {config}`
+Config could have this params:
+
+* `unwindBatchSize`:  (default 100)
+* `type`: possible values ('NONE', 'UNWIND_BATCH', 'UNWIND_BATCH_PARAMS') (default 'UNWIND_BATCH')
+
+With `NONE` it will export the file with `CREATE` statement;
+
+With 'UNWIND_BATCH` it will export the file by batching the entities with the `UNWIND` method as explained in this
+https://medium.com/neo4j/5-tips-tricks-for-fast-batched-updates-of-graph-structures-with-neo4j-and-cypher-73c7f693c8cc[article].
+
 To change the export format, you have to set it on the config params like `{format : "cypher-shell"}`.
```

Comment:
do we check that the format is cypher-shell when the user specifies unwind_params ?

Reply:
We do the check when we write the params; do you prefer it at the creation of the ExporConfig that throws an exception?

https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/90deeeb8eee4c5d3f1f7da14b10245896646b280/src/main/java/apoc/export/cypher/formatter/CreateCypherFormatter.java#L255-L285

Reply:
Probably better to check it in the config, as it might be otherwise confusing for folks, why it doesn't use params even if they specified it.